src/views/Entries/QualityControl/index.tsx

```diff
@@ -78,29 +139,30 @@ function QualityControl(props: QualityControlProps) {
                 rightContainerClassName={styles.right}
                 rightChild={(
                     <div className={styles.entryList}>
-                        { leadGroupedEntriesList.map((leadWithEntries) => {
-                            const {
-                                entries,
-                                ...leadProps
-                            } = leadWithEntries;
-
-                            const lead = { ...leadProps };
-
-                            return entries.map(e => (
-                                <EntryCard
-                                    className={styles.card}
-                                    key={e.id}
-                                    entry={e}
-                                    lead={lead}
-                                    framework={framework}
-                                    isDeleted={deletedEntries[e.id]}
-                                    onDelete={handleEntryDelete}
-                                />
-                            ));
-                        })}
+                        { pending && <LoadingAnimation /> }
+                        {response && response.results.map(e => (
+                            <EntryCard
+                                className={styles.card}
+                                key={e.id}
+                                entry={e}
+                                lead={e.lead}
+                                framework={framework}
+                                isDeleted={deletedEntries[e.id]}
+                                onDelete={handleEntryDelete}
+                            />
+                        ))}
                     </div>
                 )}
             />
+            <footer className={styles.footer}>
+                <Pager
```

Comment:
Lets use the same Pager for both pages but switch the data according to which page the user is in.
Also, for QC lets save the selected matrix header and active page / total entries count in redux.